Phaethon/InternalApi/DataManagement/LoginManagement.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Web;
+using System.Web.Security;
+using Core.Model;
+using InternalApi.DataAccess;
+using InternalApi.DataManagement.IDataManagement;
+
+namespace InternalApi.DataManagement
+{
+    public class LoginManagement : ILoginManagement
+    {
+        private readonly LoginDa _LoginDa;
+        
+        internal LoginManagement()
+        {
+            _LoginDa = new LoginDa();
+        }
+
+        public bool CreateOrUpdate(Login login)
+        {
+            try
+            {
+                using (var db = new DatabaseContext())
+                {
+                    login.Salt = GenerateSalt();
+                    login.Password =
+                        Convert.ToBase64String(ComputeHMAC_SHA256(Encoding.UTF8.GetBytes(login.Password), login.Salt));
+                    _LoginDa.CreateOrUpdate(db, login);
+                    return true;
+                }
+            }
+            catch (Exception e)
+            {
+                var whatever = e;
+                return false;
+            }
+        }
+
+        public bool Delete(int id)
+        {
+            using (var db = new DatabaseContext())
+            {
+                Login login = _LoginDa.GetLogin(db, id);
+                return _LoginDa.Delete(db, login);
+            }
+        }
+
+        public Login GetLogin(int id)
+        {
+            using (var db = new DatabaseContext())
+            {
+                return _LoginDa.GetLogin(db, id);
+            }
+        }
+
+        public int Login(string username, string password)
+        {
+            using (var db = new DatabaseContext())
+            {
+                Login login = _LoginDa.Login(db, username);
+                if (login != null)
+                {
+                    if (password != null)
+                    {
+                        if (login.Password.Equals(Convert.ToBase64String(ComputeHMAC_SHA256(Encoding.UTF8.GetBytes(password), login.Salt))))
+                        {
+                            return login.ID;
+                        }
+                        else
+                        {
+                            return 0;
+                        }
+
+                    }
+                    else
+                    { 
+                        return 0;
+                    }
+                }
+                 return 0;
+                
+            }
```

Comment:
https://github.com/RaidenRabit/Phaethon/blob/da6d6b8edb567c6e375539f3d112132bbce69993/Phaethon/InternalApi/DataManagement/LoginManagement.cs#L74-L86

You don't need those. Delete everything but line 84.